docs/python/index.rst

```diff
@@ -0,0 +1,17 @@
+
+ONNXRuntime-Extensions Python API
+==========
+
+ONNXRuntime-Extensions is a library that extends the capability of the ONNX models
+and inference with ONNX Runtime, via pre/post-processing custom operators.
+Learn more about ONNXRuntime-Extensions `here <https://github.com/microsoft/onnxruntime-extensions>`_.
+
+.. automodule:: onnxruntime_extensions
+    :members:
+    :undoc-members:
+    :private-members:
+    :special-members:
```

Comment:
consider to remove these
    :undoc-members:
    :private-members:
    :special-members:
?